ddtrace/_trace/trace_handlers.py

```diff
@@ -610,6 +611,68 @@ def _on_botocore_update_messages(ctx, span, _, trace_data, __, message=None):
     HTTPPropagator.inject(context, trace_data)
 
 
+def _on_botocore_patched_bedrock_api_call_started(ctx, request_params):
+    span = ctx[ctx["call_key"]]
+    integration = ctx["bedrock_integration"]
+    span.set_tag_str("bedrock.request.model_provider", ctx["model_provider"])
+    span.set_tag_str("bedrock.request.model", ctx["model_name"])
+    for k, v in request_params.items():
+        if k == "prompt":
+            if integration.is_pc_sampled_span(span):
+                v = integration.trunc(str(v))
+        span.set_tag_str("bedrock.request.{}".format(k), str(v))
+        if k == "n":
+            ctx.set_item("num_generations", str(v))
+
+
+def _on_botocore_patched_bedrock_api_call_exception(ctx, exc_info):
+    span = ctx[ctx["call_key"]]
+    span.set_exc_info(*exc_info)
+    prompt = ctx["prompt"]
+    integration = ctx["bedrock_integration"]
+    if integration.is_pc_sampled_llmobs(span):
+        integration.llmobs_set_tags(span, formatted_response=None, prompt=prompt, err=True)
+    span.finish()
+
+
+def _on_botocore_patched_bedrock_api_call_success(ctx, reqid, latency, input_token_count, output_token_count):
+    span = ctx[ctx["call_key"]]
+    span.set_tag_str("bedrock.response.id", reqid)
+    span.set_tag_str("bedrock.response.duration", latency)
+    span.set_tag_str("bedrock.usage.prompt_tokens", input_token_count)
+    span.set_tag_str("bedrock.usage.completion_tokens", output_token_count)
+
+
+def _on_botocore_bedrock_process_response(
+    ctx: core.ExecutionContext,
+    formatted_response: Dict[str, Any],
+    metadata: Dict[str, Any],
+    body: Dict[str, List[Dict]],
+    should_set_choice_ids: bool,
+) -> None:
+    text = formatted_response["text"]
+    dd_span = ctx[ctx["call_key"]]
+    if should_set_choice_ids:
+        for i in range(len(text)):
+            dd_span.set_tag_str("bedrock.response.choices.{}.id".format(i), str(body["generations"][i]["id"]))
+    datadog_integration = ctx["bedrock_integration"]
```

Comment:
```suggestion
    integration = ctx["bedrock_integration"]
```
Just to be consistent with the other handlers